src/core/control/Control.cpp

```diff
@@ -3,6 +3,7 @@
 #include <algorithm>   // for max
 #include <cstdlib>     // for size_t
 #include <exception>   // for exce...
+#include <filesystem>  // for pngOpen
```

Comment:
The comment should note the symbol that is used from the include, not the function in the Xournal++ code where it is used. Is `filesystem` really used somewhere?